tests/integration/frameworks/wordpress/test_wordpress_transaction_name_hooks_off.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should name WordPress web transaction as an 'Action' with the
+name generated from the template used to generate the page even when
+WordPress hooks are disabled.
+*/
+
+/*SKIPIF*/
+
+/*INI
+newrelic.framework = wordpress
+newrelic.framework.wordpress.hooks = false
+*/
+
+/*ENVIRONMENT
+REQUEST_METHOD=GET
+*/
+
+/*EXPECT_METRICS_EXIST
+Supportability/InstrumentedFunction/apply_filters
+WebTransaction/Action/template
+*/
+
+/*EXPECT_ERROR_EVENTS null */
+
+// Mock WordPress hooks; only a single callback for a given hook can be defined
+$wp_filters = [];
+
+function add_filter($hook_name, $callback) {
+  global $wp_filters;
+  $wp_filters[$hook_name] = $callback;
+}
```

Comment:
Lines 32-37 emulate [this](https://github.com/WordPress/WordPress/blob/master/wp-includes/class-wp-hook.php#L323-L327) and [this](https://github.com/WordPress/WordPress/blob/master/wp-includes/class-wp-hook.php#L87-L90).